projects/1.18/assets/electrodynamics/electrodynamics/lang/zh_cn.json

```
@@ -1 +1,555 @@
+  "tooltip.seismicscanner.oncooldown" : "冷却中",
+  "tooltip.seismicscanner.showuse" : "Shift+右击以扫描",
+  "tooltip.seismicscanner.currentore" : "当前矿石:%s",
+  "tooltip.seismicscanner.empty" : "［无］",
+  "tooltip.nightvisiongoggles.status" : "状态：",
+  "tooltip.nightvisiongoggles.on" : "开",
+  "tooltip.nightvisiongoggles.off" : "关",
+  "tooltip.jetpack.mode" : "模式：",
+  "tooltip.jetpack.moderegular" : "普通",
+  "tooltip.jetpack.modehover" : "悬停",
+  "tooltip.jetpack.modeoff" : "关闭",
+  "tooltip.countdown.tickstillcheck" : "距下次检查还有：%s刻",
+  "tooltip.seismicmarker.redstone" : "传入红石信号以显示指示线",
+  "tooltip.seismicrelay.use" : "侦测指示环并储存",
+  "tooltip.coolantresavoir.place" : "放置于采石场上方",
+  "tooltip.motorcomplex.use" : "控制采石场速度",
+  "tooltip.blockframe.joke" : "不是，你这哪来的？",
+
+"tooltip.logisticalmanager.use" : "可选：采石场存储管理",
+
+
+
+
+"death.attack.electricity" : "%s触电身亡",
+  "death.attack.accelerated_bolt" : "%1$s被%2$s的箭矢洞穿了",
+  "death.attack.accelerated_bolt_ia" : "%1$s被%2$s的箭矢洞穿了",
+  "death.attack.plasma_bolt" : "%1$s被%2$s变成了蒸汽",
+  "message.electrodynamics.guidebookclick" : "点击此处",
+  "fluid.electrodynamics.sulfuricacid" : "硫酸",
+  "fluid.electrodynamics.hydrogenfluoride" : "氢氟酸",
+  "fluid.electrodynamics.ethanol" : "乙醇",
+  "fluid.electrodynamics.polyethylene" : "聚乙烯",
+  "fluid.electrodynamics.sulfatecopper" : "硫酸铜",
+  "fluid.electrodynamics.sulfatetin" : "硫酸锡",
+  "fluid.electrodynamics.sulfatesilver" : "硫酸银",
+  "fluid.electrodynamics.sulfatelead" : "硫酸铅",
+  "fluid.electrodynamics.sulfateiron" : "硫酸铁",
+  "fluid.electrodynamics.sulfategold" : "硫酸金",
+  "fluid.electrodynamics.sulfatevanadium" : "硫酸钒",
+  "fluid.electrodynamics.sulfatelithium" : "硫酸锂",
+  "fluid.electrodynamics.sulfatemolybdenum" : "硫酸钼",
+  "fluid.electrodynamics.sulfatenetherite" : "硫酸卡拜",
+  "fluid.electrodynamics.concrete" : "液态混凝土",
+  "fluid.electrodynamics.clay" : "液态黏土",
+  "fluid.electrodynamics.oxygen" : "氧气",
+  "fluid.electrodynamics.hydrogen" : "氢气",
+  "fluid.electrodynamics.hydraulic" : "液压油",
+  "subtitles.electrodynamics.hum" : "噪声",
+  "subtitles.electrodynamics.mineralcrusher" : "矿石粉碎机：运行",
+  "subtitles.electrodynamics.mineralgrinder" : "矿石研磨机：运行",
+  "subtitles.electrodynamics.electricpump" : "电动泵：运行",
+  "subtitles.electrodynamics.windmill" : "扇叶摩擦空气",
+  "subtitles.electrodynamics.combustionchamber" : "燃烧室：运行",
+  "subtitles.electrodynamics.ceramicplatebreaking" : "陶瓷板：破坏",
+  "subtitles.electrodynamics.ceramicplateadded" : "陶瓷板：安装",
+  "subtitles.electrodynamics.kineticrailgun" : "动能轨道炮：开火",
+  "subtitles.electrodynamics.kineticrailgun.noammo" : "动能轨道炮：未就绪",
+  "subtitles.electrodynamics.plasmarailgun" : "等离子轨道炮：开火",
+  "subtitles.electrodynamics.kineticrailgun.nopower" : "等离子轨道炮：电力不足",
+  "subtitles.electrodynamics.rodhittingground" : "轨道炮弹丸：击中地面",
+  "subtitles.electrodynamics.latherunning" : "车床：运行",
+  "subtitles.electrodynamics.equipheavyarmor" : "复合装甲防具：铿锵",
+  "subtitles.electrodynamics.hydroelectricgenerator" : "水轮旋转",
```

`Hydroelectric generator spins`
```suggestion
  "subtitles.electrodynamics.hydroelectricgenerator" : "水力发电机：旋转",
```